Cargo.toml

```diff
@@ -6,3 +6,11 @@ lto = "fat"
 codegen-units = 1
 opt-level = "z"
 debug = true
+
+[patch.crates-io]
+cosm-orc = { git = "https://github.com/elys-network/cosm-orc.git", branch = "dependencies-fix" }
+cosm-tome = { git = "https://github.com/elys-network/cosm-tome.git", branch = "dependencies-fix", features = [
```

Comment:
@politeWall those changes got merged into those repos’s main branches, so I believe the branch value here should be `main` for the 2 lines here right?